src/pages/kyunghyun/Login/Login.js

```diff
@@ -1,8 +1,174 @@
 import React from 'react';
-
+// import { Link } from 'react-router-dom';
+import './Login.scss';
+import '../../../styles/common.scss';
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+      alert: '',
+      valid: false,
+    };
+  }
+
+  handleInput = e => {
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  }; // id/pw 통합 input핸들러 : state객체 속성을 이용 []:value 방식으로 적용되는 input을 나눔
+
+  checkValid = e => {
+    const { id, pw } = this.state;
+    const validId = id.includes('@');
+    const validPw = pw.length >= 6;
+    validId && validPw
+      ? this.setState({ valid: true })
+      : this.setState({ valid: false });
```

Comment:
```suggestion
    this.setState({ isLoginValid: isValidId && isValidPw })
```
이렇게 적어주더라도 동일한 표현이네요